README.md

```diff
@@ -14,32 +14,77 @@ Playwright is a framework for Web Testing and Automation. It allows testing [Chr
 
 Headless execution is supported for all the browsers on all platforms. Check out [system requirements](https://playwright.dev/docs/library#system-requirements) for details.
 
-## Usage
+## Installation
 
+Playwright has its own test runner for end-to-end tests, we call it Playwright Test.
+
+### Using init command
+
+The easiest way to get started with Playwright Test is to run the init command.
+
+```Shell
+# Run from your project's root directory
+npm init playwright
+# Or create a new project
+npm init playwright new-project
 ```
-npm i -D playwright
+
+This will create a configuration file, optionally add examples, a GitHub Action workflow and a first test example.spec.ts. You can now jump directly to writing assertions section.
+
+### Manually
+
+Add dependency and install browsers.
+
+```Shell
+npm i -D @playwright/test
+# install supported browsers
+npx playwright install
 ```
 
```

Comment:
At this point we should redirect users to our official docs /docs/intro.

Reply:
Had chat offline. Resolving.